### ISSUE TITLE
`pmdk-convert` doesn't support PMDK minor versions >=10 (PMDK v1.10 or later)

### ISSUE THREAD

**Issue by sscargal** [pmem/pmdk-convert#92]
Since PMDK v1.10 is now available, `pmdk-convert` should be updated to support two-digit minor versions to avoid the following error:

```
# /usr/local/bin/pmdk-convert --from=1.9 --to=1.10 /dev/dax0.0
Invalid "to" version format [major.minor].
Usage: pmdk-convert [--version] [--help] [--no-warning] --from=<version> --to=<version> <pool>
```

The pool layout created by PMDK v1.10 using `pmempool create` creates a pool major version = 6. However, `pmdk-convert` should still support PMDK v1.10 and later. 

The error originates from `conv_version()` that expects X.Y versions, not X.YZ due to the length check on [line 293](https://github.com/pmem/pmdk-convert/blob/master/pmdk-convert.c#L293).

```
static int
conv_version(const char *strver)
{
	if (strlen(strver) != 3)
		return -1;
```

**Comment by pbalcer** [comment]
Thanks, good catch.
The layout hasn't changed, so as a workaround it's possible to just use `--to=1.9` or `--to-layout=6`.

**Comment by pbalcer** [comment]
fixed in #93
